dump2polarion/ostriztools.py

```diff
@@ -47,13 +47,9 @@ def _get_testrun_id(version):
     """Gets testrun id out of the appliance_version file."""
     try:
         build_base = version.strip().split("-")[0].split("_")[0].replace(".", "_")
-        zval = int(build_base.split("_")[3])
```

Comment:
Discussed offline with @mkoura, using `packaging.version.Version` at his suggestion instead of introducing a requirement for miq_version.  This will also keep things a bit more abstract, and not as focused on CFME.